tests/frame/schema_test.py

```diff
@@ -89,7 +90,8 @@ class Foo: ...
 
 
 @pytest.mark.skipif(PANDAS_VERSION < (2, 0, 0), reason="too old")
-def test_dtypes() -> None:
+@pytest.mark.parametrize("nw", [nw_main, nw])
+def test_dtypes(nw: ModuleType) -> None:
```

Comment:
i don't think we need to parametrise these, can we can do the v1-specific test in the new `v1_test.py`